nowcasting_api/tests/test_status.py

```diff
@@ -25,7 +26,8 @@
 client = TestClient(app)
 
 
-def test_read_latest_status(db_session):
+@pytest.mark.asyncio
+async def test_read_latest_status(db_session):
```

Comment:
why does this have to be async?

Reply:
As a precaution, I had changed it from synchronous to asynchronous, but now it works.